flang/lib/Optimizer/CodeGen/TypeConverter.cpp

```diff
@@ -156,8 +146,8 @@ std::optional<llvm::LogicalResult> LLVMTypeConverter::convertRecordType(
     else
       members.push_back(mlir::cast<mlir::Type>(convertType(mem.second)));
   }
-  if (mlir::failed(st.setBody(members, /*isPacked=*/false)))
-    return mlir::failure();
+  auto st = mlir::LLVM::LLVMStructType::get(&getContext(), name, members,
```

Comment:
```suggestion
  auto structType = mlir::LLVM::LLVMStructType::get(&getContext(), name, members,
```
ultra nit: I would spell out